infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/server/RouteServerExposer.java

```diff
@@ -91,6 +96,17 @@
  */
 @Singleton
 public class RouteServerExposer implements ExternalServerExposer<OpenShiftEnvironment> {
+
+  private final Map<String, String> labels;
+
+  public RouteServerExposer(
+      @Nullable @Named("che.infra.openshift.route.labels") String labelsProperty) {
+    this.labels =
+        labelsProperty != null
+            ? Splitter.on(",").withKeyValueSeparator("=").split(labelsProperty)
```

Comment:
As above, IMHO, handling extraneous whitespace is desirable.

Reply:
Splitter is taking care of trims.